staking/integration-tests/src/setup.rs

```diff
@@ -115,7 +116,7 @@ pub fn setup(props: SetupProps) -> SetupResult {
         .publishers
         .iter()
         .position(|&x| x == publisher_keypair.pubkey())
-        .unwrap();
+        .unwrap_or(0);
```

Comment:
This seems wrong?

Reply:
Maybe it's better to make publisher_index an `Option`

Reply:
Done